src/WebApiCompetitionService/Applitcation/Controllers/EquipesUsuariosController.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebAPICompetitionService.Applitcation.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
Po, acho que no nosso padrão seria sem o api, só o controller, confere @VictorGM01 ? Ou coloca API? Se for colocar mudo no outro